app-frontend/src/types/initialState.ts

```diff
@@ -3,12 +3,16 @@ import { Worker } from './worker';
 
 export interface InitialState {
```

Comment:
The same story here.
1. This is not type of initial state, this is AppState
2. App state should be composed from individual reducers types